security/entity_provider.rst

```diff
@@ -42,12 +42,14 @@ with the following fields: ``id``, ``username``, ``password``,
 
     use Doctrine\ORM\Mapping as ORM;
     use Symfony\Component\Security\Core\User\UserInterface;
+    use Serializable;
```

Comment:
This should be reverted. We follow the Symfony code style which does not add `use` statements for classes from the global namespace.